prow.go

```diff
@@ -408,7 +411,7 @@ func (m *jobManager) newJob(job *Job) error {
 
 	data, _ := json.MarshalIndent(sourceConfig, "", "  ")
 	klog.V(2).Infof("Found target job config %s/%s:\n%s", srcNamespace, srcName, string(data))
-	prow.OverrideJobEnvVar(&pj.Spec, "CONFIG_SPEC", string(data))
+	prow.OverrideJobEnvVar(&pj.Spec, "UNRESOLVED_CONFIG", string(data))
```

Comment:
You also need to override config spec to set it to nil.

Reply:
Done